package.json

```diff
@@ -66,10 +66,10 @@
   },
   "dependencies": {
     "object.assign": "^4.1.0",
-    "react-with-styles-interface-aphrodite": "^5.0.0"
+    "react-with-styles-interface-aphrodite": "^6.0.0"
```

Comment:
This major version includes `react-with-styles@4` as a peer dep as well.

Reply:
if the interface requires 4, not 3 || 4, then this is a major change, and the below peer dep should be just 4 as well

Reply:
@ljharb The interface has 3 || 4. 🤔 So this should be good.

Reply:
It was a breaking change because of the drop of node < 6